gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPluginExtension.groovy

```diff
@@ -21,6 +15,11 @@ class AkkaGrpcPluginExtension {
     boolean generatePlay = false
     boolean serverPowerApis = false
     boolean usePlayActions = false
+    boolean includeStdTypes = false
+    String protocVersion = "3.11.4"
+    String grpcVersion = "1.32.1"
+    String protocPluginScalaVersion = "2.12"
```

Comment:
I wonder if we can do even better, and instead of referring to constants here, refer to the generated `akka.grpc.codegen.BuildInfo` class here.

Is this refactor needed for this PR? If not it might be better to do it in a separate one.

Reply:
I'd want to refer to the BuildInfo, but do not know the way to get it done. Perhaps, a code-generation task would help. Unfortunately, my sbt knowledge is not sufficient for this task.

I do need the 'includeStdTypes' variable. I will revert other changes.